requirements-dev.txt

```diff
@@ -7,6 +7,7 @@ docker>=6
 time-machine>=2.13.0
 honcho>=1.1.0
 openapi-core>=0.18.2
+openapi-pydantic==0.4.0
```

Comment:
requirements-dev minimal file should not `==` these should all be `>=`

Reply:
IIRC, I made this `==` because a later version has a compatibility issue with our version of the `pydantic` dependency. We'd have to upgrade both, and I believe there are downstream challenges from upgrading Pydantic.

Is it okay to make an exception for that?

Reply:
we should try and get everything upgraded with some urgency then -- if we can't upgrade `pydantic` for a security fix we're pretty stuck!

you should still be able to set this to `>=` and the freezing won't bump it unless necessary

Reply:
I'll investigate.

Reply:
I tried to reproduce whatever compatibility error I encountered originally and didn't find anything. Upgrading Pydantic to 2.x would be nice, but AFAICT is not a blocker for this feature. I've changed the dep to `>=`.